pallets/duster/src/tests.rs

```diff
@@ -226,41 +202,20 @@ fn native_existential_deposit() {
 				amount: 600,
 			}
 			.into(),
-			orml_currencies::Event::Transferred {
```

Comment:
shouldn't these events be back with our custom pallet?

Reply:
Yes, but I removed them because I believe it's enough to test `Balances`/`Tokens` events.